packages/functional-tests/.eslintrc.js

```diff
@@ -4,14 +4,19 @@ module.exports = {
     browser: true,
   },
   extends: [
-    // 'eslint:recommended',
-    // 'plugin:playwright/recommended',
+    'eslint:recommended',
+    'plugin:playwright/recommended'
   ],
+  globals: {
+    globalThis: false, // not writeable
+  },
   rules: {
     '@typescript-eslint/no-unused-vars': 'warn',
     '@typescript-eslint/ban-ts-comment': 'off',
     '@typescript-eslint/ban-types': 'warn',
-    // 'playwright/missing-playwright-await': 'off',
-    'no-console': ['error', { allow: ['log'] }], // console errors cause side effects in CircleCI
+    'playwright/missing-playwright-await': 'off',
```

Comment:
Not sure why, but this was throwing a hard error somewhere inside Playwright about not being able to find `parent`? 🤷 
Turning off for now and we can troubleshoot "later".